example/bufferCopy/CMakeLists.txt

```diff
@@ -53,18 +53,25 @@ SET(_SOURCE_DIR "src/")
 # Add all the source files in all recursive subdirectories and group them accordingly.
 append_recursive_files_add_to_src_group("${_SOURCE_DIR}" "${_SOURCE_DIR}" "cpp" _FILES_SOURCE)
 
+SET(_COMPILE_OPTIONS_PRIVATE ${ALPAKA_DEV_COMPILE_OPTIONS})
+
 IF(MSVC)
     LIST(APPEND _COMPILE_OPTIONS_PRIVATE "/wd4996")   # This function or variable may be unsafe. Consider using <safe_version> instead.
 ENDIF()
 
+# NVCC does not incorporate the COMPILE_OPTIONS of a target but only the CMAKE_CXX_FLAGS
+IF(ALPAKA_ACC_GPU_CUDA_ENABLE AND NOT ALPAKA_CUDA_COMPILER MATCHES "clang")
+    STRING(REPLACE ";" " " _COMPILE_OPTIONS_PRIVATE_STRING "${_COMPILE_OPTIONS_PRIVATE}")
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_COMPILE_OPTIONS_PRIVATE_STRING}")
+ENDIF()
```

Comment:
As was noted recently in one of the issues, `TARGET_COMPILE_OPTIONS` does not work when nvcc is used. Therefore, the `COMPILE_OPTIONS` have to be added to the `CMAKE_CXX_FLAGS` before calling `CUDA_ADD_EXECUTABLE`.
This is not a problem on linux because there the additional options only contain some enhanced warning options. However, on windows the `/bigobj` switch is required for the build to succeed.
With this change, the enhance warning options will also be used for builds with nvcc.

Reply:
On windows you can set `/bigobj` generally, you need it all the time and it's just not set by default for some backwards compatibility reasons with an ancient windows/msvc version.